R/callback.R

```diff
@@ -26,3 +36,22 @@ build_callback.luz_module_generator <- function(object,
 
   luz_callback()
 }
+
+build_callback.keras.engine.training.Model <- function(object,
+                                                       type = c("l1_norm", "l2_norm")) {
+  keras::`%py_class%`(
+    keras_callback(keras::keras$callbacks$Callback), {
+      initialize <- function() {
+        super$initialize()
+        self$constraint <- keras_constraint(norm_order(type))
+      }
+
+      on_train_batch_end <- function(batch, logs = NULL) {
+        for (layer in utils::head(self$model$layers, -1))
+          self$constraint(layer)
```

Comment:
@moralapablo If I'm not mistaken, we should check and constraint the last layer in some cases here (what would be the check?, when there's no activation?). This works for luz because neurons and activation functions are separate layers, so in luz we always want to avoid the last layer.

Reply:
It's ok.